### ISSUE TITLE
[Feature request] MFA/2FA/Google Authentificator

### ISSUE THREAD

**Issue by sadortun** [repman-io/repman#325]
Hi!

This is a quick request :) 

Would it be possible to add Google Authenticator ? ( https://github.com/scheb/2fa )

Have a good day, 
Samuel

**Comment by akondas** [comment]
Yes, it looks like a fairly simple integration. I am not in a position to deal with it at the moment, but I can take a look at it next week. Thanks for the link :wink:
